.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+## Description
+<!-- Add a brief and meaningful description. -->
+
+## Expected Behavior
+<!-- Describe the expected behaviour. -->
+
+## Actual Behavior
+<!-- Describe the current/actual behaviour. -->
+
+## Environment
+
+* Operating system: (E.g RHEL 7.6 )
+* OpenShift version:
+<!-- Run the command `oc version` and add the result here. -->
+* Ansible version:
+<!-- Run the command `ansible --version` and add the result here. -->
+* Project Version/Tag: (E.g release-1.0.1)
```

Comment:
Hi @tremes and @trepel,

Could you help here? Is this env good for this project? Could you review this PR?

Reply:
I think it is OK @camilamacedo86

Reply:
@camilamacedo86 well I think the environment (and likely other items) description are not necessary here...or simply doesn't make sense. The jobs generated from this repository will always run in a Jenkins and I don't think there will be some releases.